front/tests/007-op-rollingstock-tab.spec.ts

```diff
@@ -1,62 +1,42 @@
 import { expect, test } from '@playwright/test';
-import { v4 as uuidv4 } from 'uuid';
-
-import type {
-  Infra,
-  Project,
-  RollingStock,
-  Scenario,
-  Study,
-  TimetableResult,
-} from 'common/api/osrdEditoastApi';
-
-import scenarioData from './assets/operationStudies/scenario.json';
+
+import type { Project, RollingStock, Scenario, Study } from 'common/api/osrdEditoastApi';
+
 import OperationalStudiesPage from './pages/operational-studies-page-model';
 import RollingStockSelectorPage from './pages/rollingstock-selector-page';
 import ScenarioPage from './pages/scenario-page-model';
-import { getProject, getStudy, getRollingStock, postApiRequest, getInfra } from './utils/api-setup';
+import { getRollingStock } from './utils/api-setup';
+import setupScenario from './utils/scenario';
 
-let smallInfra: Infra;
 let project: Project;
 let study: Study;
 let scenario: Scenario;
 let rollingStock: RollingStock;
-let timetableResult: TimetableResult;
 
 const dualModeRollingStockName = 'dual-mode_rollingstock_test_e2e';
 const electricRollingStockName = 'rollingstock_1500_25000_test_e2e';
 test.beforeAll(async () => {
-  smallInfra = (await getInfra()) as Infra;
-  project = await getProject();
-  study = await getStudy(project.id);
   rollingStock = await getRollingStock();
 });
 
-test.beforeEach(async () => {
-  timetableResult = await postApiRequest(`/api/timetable/`);
-  scenario = await postApiRequest(`/api/projects/${project.id}/studies/${study.id}/scenarios/`, {
-    ...scenarioData,
-    name: `${scenarioData.name} ${uuidv4()}`,
-    study_id: study.id,
-    infra_id: smallInfra.id,
-    timetable_id: timetableResult.timetable_id,
-  });
+test.beforeEach(async ({ page }) => {
+  const scenarioPage = new ScenarioPage(page);
+
+  ({ project, study, scenario } = await setupScenario());
```

Comment:
Why do you create a scenario for each test ? I think 1 scenario is enough since you don't modify the scenario.
If 1 scenario is enough, then you can move this line in the beforeAll

Reply:
The tests are run in parallel, so we can’t make changes to the same scenario like selecting a rolling stock (RS) or modifying it without causing conflicts

Reply:
This may be worth a comment.

Reply:
Actually, changes like the selection of the rolling stock are managed in the navigator and do not impact the scenario in the database, that is why I think 1 scenario is enough. Have you tried with only 1 scenario ?